dev/TeachingTip/TeachingTip.cpp

```diff
@@ -783,18 +808,26 @@ void TeachingTip::OnTargetOffsetChanged()
 
 void TeachingTip::OnIsLightDismissEnabledChanged()
 {
-    if (m_popup)
-    {
-        m_popup.get().IsLightDismissEnabled(IsLightDismissEnabled());
-    }
-
     if (IsLightDismissEnabled())
     {
         winrt::VisualStateManager::GoToState(*this, L"LightDismiss"sv, false);
+        if (auto lightDismissIndicatorPopup = m_lightDismissIndicatorPopup.get())
+        {
+            lightDismissIndicatorPopup.IsLightDismissEnabled(true);
+            m_lightDismissIndicatorPopupClosedRevoker = lightDismissIndicatorPopup.Closed(winrt::auto_revoke, { this, &TeachingTip::OnLightDismissIndicatorPopupClosed });
```

Comment:
This code looks duplicated with the above. Any chance of having this codepath be leveraged from OnIsOpenedChanged?